docs/upgrading-to-65.asciidoc

```diff
@@ -119,6 +119,7 @@ The data format remains unchanged.
 The Metricsets endpoint, previously `/v1/metrics`,
 has also integrated with the new intake endpoint at `/intake/v2/events`.
 In terms of functionality, the only change relates to timestamps.
+The `metrics.enabled` setting only applies to `v1/metrics`, metricsets can't be disabled in v2.
```

Comment:
@bmorelli25 we had this setting for the metrics v1 endpoint, but in v2 is not used because metrics go through the general intake endpoint and can't be disabled.

We missed it this back in 6.5, trying to fix it now. Is this a good way and the right place to note?

Reply:
This is the right place, but I'm concerned that the line above this says "the only change relates to timestamps". I'm thinking the paragraph (L119-123) should be changed to something like this:

```
The Metricsets endpoint, previously `/v1/metrics`,
has also integrated with the new intake endpoint at `/intake/v2/events`.
There are two changes to be aware of:

* Timestamps are now UTC based and formatted as microseconds since Unix epoch.
* The `metrics.enabled` setting has been deprecated and only applies to `v1/metrics`.
Metricsets can't be disabled in v2.

See the <<metricset-api, metricset>> schema for more information.
```

Reply:
thank you, updated :+1: